### PR TITLE
update: More detailed Session description

### PR DESCRIPTION
**Pull request by jermanuts**
Changes proposed in this PR:

Explain how Session open group works. Current one is vague and lacking.

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/coc).

### CODE REVIEW

**Comment by jermanuts** [comment]
Should we recommend also disabling Link previews as mentioned here https://discuss.privacyguides.net/t/critical-flaws-in-desktop-session-messenger/16674 ?

**Review by dngray — APPROVED** [review]

**Review by dngray — COMMENTED** [review]
Wouldn't start a sentence with "however".

**Review by dngray — CHANGES_REQUESTED** [review]

**Review by jonaharagon — CHANGES_REQUESTED** [review]
> However, they are not as private as one-on-one chats or closed groups.

can we link to a resource which explains why this is the case in more detail? or explain why ourselves

**Comment by dngray** [comment]
> can we link to a resource which explains why this is the case in more detail? or explain why ourselves

that might be a better idea.

**Comment by jermanuts** [comment]
I got it from https://getsession.org/lightpaper/pdf. Their docs pretty lacking tbh.

File: docs/real-time-communication.md [hunk 1]
```diff
@@ -202,7 +202,8 @@ Session uses the decentralized [Oxen Service Node Network](https://oxen.io) to s
 
 </div>
 
-Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. Open groups have no restriction on the number of members, but are open by design.
+Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. Open groups have no restriction on the number of members, they are hosted on federated servers operated by the communities themselves, and moderation policies are determined by each individual community. However, they are not as private as one-on-one chats or closed groups.
```
Comment (jonaharagon) [inline docs/real-time-communication.md:205]:
```suggestion
Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. It is also possible to [host](https://docs.oxen.io/oxen-docs/products-built-on-oxen/session/guides/open-group-setup) or join an open group which can host thousands of members, but messages in these open groups are **not** end-to-end encrypted between participants.
```

**Review by jonaharagon — CHANGES_REQUESTED** [review]

File: docs/real-time-communication.md [hunk 2]
```diff
@@ -251,7 +251,8 @@ Session uses the decentralized [Oxen Service Node Network](https://oxen.io) to s
 
 </div>
 
-Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. Open groups have no restriction on the number of members, but are open by design.
+Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. It is also possible to [host](https://docs.oxen.io/oxen-docs/products-built-on-oxen/session/guides/open-group-setup) or join an open group which can host thousands of members, but messages in these open groups are **not** end-to-end encrypted between participants.
+
```
Comment (friadev) [inline docs/real-time-communication.md:255]:
```suggestion
```

**Review by friadev — CHANGES_REQUESTED** [review]

**Review by friadev — APPROVED** [review]

**Review by redoomed1 — DISMISSED** [review]
Just a tiny suggestion to change wording so that it reads a little smoother but otherwise LGTM

File: docs/real-time-communication.md [hunk 3]
```diff
@@ -251,7 +251,7 @@ Session uses the decentralized [Oxen Service Node Network](https://oxen.io) to s
 
 </div>
 
-Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. Open groups have no restriction on the number of members, but are open by design.
+Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. It is also possible to [host](https://docs.oxen.io/oxen-docs/products-built-on-oxen/session/guides/open-group-setup) or join an open group which can host thousands of members, but messages in these open groups are **not** end-to-end encrypted between participants.
```
Comment (redoomed1) [inline docs/real-time-communication.md:254]:
```suggestion
Session allows for E2EE in one-on-one chats or closed groups which allow for up to 100 members. It is also possible to [set up](https://docs.oxen.io/oxen-docs/products-built-on-oxen/session/guides/open-group-setup) or join open groups which can host thousands of members, but messages in these open groups are **not** end-to-end encrypted between participants.
```

**Review by redoomed1 — APPROVED** [review]
Just a tiny suggestion to change wording so that it reads a little smoother but otherwise LGTM

**Review by jonaharagon — APPROVED** [review]

**Pull request merged**